### PR TITLE
BWAA-TV RC4

### PR DESCRIPTION
**Pull request by dbpokorny**
Leaving media.hbs as it is because it has diverged substantially from main.hbs

### CODE REVIEW

**Comment by botwillacceptanything** [comment]
#### :ballot_box_with_check: Voting procedure reminder:
To cast a vote, post a comment containing `:+1:` (:+1:), or `:-1:` (:-1:).
Remember, you **must :star:star this repo for your vote to count.**

All comments within this discussion are searched for votes, regardless of the time of posting.
You can cast as many votes as you want, but only the last one will be counted.
(You may consider editing your comment instead of adding a new one.)
Comments containing both up- and down-votes are disregarded.
PR authors automatically count as a :+1: vote.

A decision will be made after this PR has been open for **15** minutes (plus/minus **10** percent, to avoid people timing their votes), and at least **8** votes have been made.
A supermajority of **65%** is required for the vote to pass.

*NOTE: the PR will be closed if any new commits are added after:* 14fa69eed26b12a1f256defa42f8aff8c0de2ac9

**Comment by SpenserJ** [comment]
Still not liking the idea of CSS defined in the page template, but :+1: because I want to see this happen

File: data/views/layouts/media.hbs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        .first {
+            margin-top: 0px;
+            padding-top: 0px;
+        }
+
+        .boxed {
+            border: 1px solid black;
+        }</style>
+    </head>
+    <body onload="pageLoaded()"><h1 id="Header">BWAA TV - The project where anything goes</h1>
```
Comment (shalecraig) [inline data/views/layouts/media.hbs]: Please use jquery, instead of an `onload=...` attribute.

```javascript
$(function() {
... content here ...
});

```

**Comment by dbpokorny** [comment]
These are small refactorings, and I'll add theses as fixes in a separate PR after this goes live.

**Comment by PurHur** [comment]
:-1: Oh no. Dont give the anythingbot to the datacrawler army.

**Comment by samis** [comment]
:+1:  This is good.

**Comment by pandzieciol** [comment]
:+1:

**Comment by dbpokorny** [comment]
test it out: http://45.33.44.57:3000/video/

**Comment by botwillacceptanything** [comment]
#### :warning: This PR has been modified and is now being closed.

To prevent people from sneaking in changes after votes have been made, pull requests can't be committed on after they have been opened. Feel free to open a new PR for the proposed changes to start another round of voting.

File: lib/media.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    }
+  };
+
+  var media = {
+    getMediaAt : getMediaAt,
+    items : items,
+    grandTotalSeconds : grandTotalSeconds,
+  }
+
+  module.exports = media;
```
Comment (shalecraig) [inline lib/media.js]: Can you inline the intermediate `media` variable?

**Pull request closed without merging**